submissions/fabunmiTolulope/HTML/Productpage.html

```diff
@@ -4,54 +4,86 @@
-	<h3>Similar breeds of banana available</h3>
-	<ol>
-		<li type="i">Plantain (Cooking banana) </li>
-		<li type="i"> Grand Nain banana</li>
-		<li type="i"> Gros michel banana</li>
-		<li type="i"> Lady finger banana</li>
-		<li type="i"> <a href="#">  More items </a></li>
-	</ol> 
-</p><hr>
+<div class="description">
+	<h3>DESCRIPTION </h3>
+	<p>From a cultivars that dominate the international trade in bananas, this <strong>Organic giant Cavendish Banana (<em>Musa sapenticum</em>)</strong> sets the standards in terms of taste and nutrient yield. <br>They are rich in Vitamin C, B6, minerals and dietary fibres. They are a rich energy source with 25%  carbohydrate. <br> It could be used for Banana Starch, flour and chips.<br> They can be eaten raw, used as smoothie and added to your homemade yummy yoghurt and Icecream. </p>
+</div>
+<section class="similar">
+	<h3>SIMILAR BREEDS OF BANANA</h3>
+		<ul>
+			<li> 
+				<figure>
+				<img src="../html/images/plantain.jpg">
+				<figcaption>Plantain (Cooking banana). <br> Price: &#x20A6;10,000</figcaption>
+				</figure>
+			</li>
+			<li>
+				<figure>
+				<img src="../html/images/gros-michel.jpg">
+				<figcaption> Gros michel banana <br> Price: &#x20A6;10,500 </figcaption>
+				</figure>
+			</li>
+			<li>
+				<figure>
+				<img src="../html/images/lady-finger.jpg">
+				<figcaption> Lady finger banana <br> Price: &#x20A6;8,500 </figcaption>	
+				</figure>
+			</li>
+		</ul> 
 </section>
-<nav>
-	<ul>
-		<li><a href="#"> Add to cart</a></li>
-		<li><a href="#">View more products</a> </li>
-	</ul>
-</nav>
 <footer>
-	<p> For more enquiries: <em>contact us on +2347063740223 or send a mail to us at Greenprint@gmail.com</em>
-		<ul>
-			<li><a href="https://wwww.Instagram.com">Instagram</a></li>
-			<li><a href="https://wwww.facebook.com">facebook</a></li>
-		</ul>
-	</p>
-	<p><big>&copy; 2020 Greenprint Integrated Services. Copyrights reserved</big></p>
+    <div class="footer1">
+    	<div class="footer2">
+       		<div class="footerlist">
+        		<ul>
+            		<li><a href="../html-forms/contact-form.html">Contact Us</a></li>
+            		<li><a href="#">About Us</a></li>
+            		<li><a href="#">Terms and Conditions</a></li>
+        		</ul>
+       		</div>
+      		<div class="footer-signup">
+        		<h4>Sign up for our Newsletter</h4>
+          		<input type="email" id="footer-email" name="email" placeholder="Enter your email address">
+          		<input type="button" id="footer-submit" value="SUBMIT">
+       		</div>
+    		<div class="footer-connect">
+        		<h4>CONNECT WITH US</h4>
+    			<a href="https://www.facebook.com"><input type="image" class="img" name="facebook" src="../html/images/facebook-logo.jpg"></a>
```

Comment:
The images should not be an input field. It should be a `img` tag. @ToluEnny24

Reply:
this has been effected @ToluAkin